projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts

```diff
@@ -384,6 +393,19 @@ describe('dot-key-value', () => {
                 });
             });
         });
+
+        describe('statusChange', () => {
+            it('should emit on lost focus in autocomplete', async () => {
+                const form = await getForm();
+                form.triggerEvent('lostFocus', {});
```

Comment:
TBH, not sure if this needs `await`

Reply:
just checked not needed, so all test code that uses `triggerEvent` does not need it, and we need to remove it!